tbf-challenge/teamwork-be
# Ft get articles endpoint 57

#### Pre-Submission Checklist






#### Checklist:





#### What does this PR do?



#### How should this be manually tested?


1. Start the server with `npm start`
2. Make a get request to this endpoint  ` http://localhost:3000/api/v1/feed `
3. This should successfully return all articles in the database
4. Press send 

##### Any background context you want to provide?


This endpoint gets all the existing articles in the database

#### What are the relevant issues?
 This endpoint is for retrieving all articles in the database


#### Screenshots (if appropriate)



![Screenshot from 2022-08-15 12-38-14](https://user-images.githubusercontent.com/68056104/184636737-fded1e8a-a538-496f-bc8c-14aac53f5faa.png)

#### Does it require any deployments updates ?
Yes it does, it is a work in progress

https://github.com/tbf-challenge/teamwork-be/issues/57

Yes, it does
#### Questions:
